diff --git a/ta1_server/backend/alignment.py b/ta1_server/backend/alignment.py
index 5148f00..df94b23 100644
--- a/ta1_server/backend/alignment.py
+++ b/ta1_server/backend/alignment.py
@@ -131,8 +131,8 @@ def convert_to_alignment_results(package: ml.AlignmentPackage,
     ## Extract KDMA scores
     rdm = package.rdm_alignments[0] # For now we are only using one RDM
     kdma_values = []
-    for kdma_id, kdma_value in rdm.alignment_detail.kdma_alignments.items():
-        kdma_values.append(ps.KDMA(kdma=kdma_id, value=kdma_value))
+    for kdma_id, kdma_value in package.aligner_profile.kdma_measurements.items():
+        kdma_values.append(ps.KDMA(kdma=kdma_id, value=kdma_value.value))
 
     ## The AlignmentResults object
     result = ps.AlignmentResults(

