diff --git a/ta1_server/backend/alignment.py b/ta1_server/backend/alignment.py
index 5148f00..1cf9ef3 100644
--- a/ta1_server/backend/alignment.py
+++ b/ta1_server/backend/alignment.py
@@ -131,7 +131,7 @@ def convert_to_alignment_results(package: ml.AlignmentPackage,
     ## Extract KDMA scores
     rdm = package.rdm_alignments[0] # For now we are only using one RDM
     kdma_values = []
-    for kdma_id, kdma_value in rdm.alignment_detail.kdma_alignments.items():
+    for kdma_id, kdma_value in package.aligner_profile.kdma_measurements.items():
         kdma_values.append(ps.KDMA(kdma=kdma_id, value=kdma_value))
 
     ## The AlignmentResults object
